AUTHORS.rst

```diff
@@ -52,6 +52,7 @@ Contributors
 * Hugo van Kemenade -- support FORCE_COLOR and NO_COLOR
 * Ian Lee -- quickstart improvements
 * Jacob Mason -- websupport library (GSOC project)
+* James Addison -- fixes and test maintenance for linkcheck and HTML search
```

Comment:
Don't undersell the work! It isn't just 'fixes'...
```suggestion
* James Addison -- linkcheck and HTML search
```

Reply:
Mmm.. I'd prefer not to imply that I developed them in the first place, though.  `Contributions to linkcheck and HTML search`?

Reply:
If you feel uncomfortable with that, how about "Major contributions to" because your work is non-trivial.

Reply:
Thanks for the compliment, but that doesn't seem quite right to me either.  I'll go for `linkcheck and HTML search improvements` - and will adjust that in a few minutes.